docker/release/dockerfiles/opensearch-dashboards.al2.dockerfile

```diff
@@ -93,5 +98,5 @@ LABEL org.label-schema.schema-version="1.0" \
   org.label-schema.build-date="$BUILD_DATE"
 
 # CMD to run
-ENTRYPOINT ["./opensearch-dashboards-docker-entrypoint.sh"]
+ENTRYPOINT ["tini", "--", "./opensearch-dashboards-docker-entrypoint.sh"]
```

Comment:
Is it possible to use ENTRYPOINT + CMD together?
```

ENTRYPOINT ["/tini", "--"]

# Run your program under Tini
CMD ["/your/program", "-and", "-its", "arguments"]
```

Reply:
Not sure about this since this may change behavior existing invocations, but AFAICS non-`opensearch` command invocations will just exec the supplied command with a couple of env var `exports`

Reply:
Seems like you add this now?